src/applications/vaos/tests/e2e/covid19-vaccine.cypress.spec.js

```diff
@@ -242,7 +242,7 @@ describe('VAOS COVID-19 vaccine appointment flow', () => {
     cy.axeCheckBestPractice();
   });
 
-  it.skip('should show facility contact page when vaccine schedule is not available', () => {
+  it('should show facility contact page when vaccine schedule is not available', () => {
```

Comment:
This test was broken before it started failing and was always showing an error message. But for some reason last week it started not changing the url and therefore started failing. I've fixed the error and added a check to make sure some of the content on the page exists.